wiki/content/migration/migrate-dgraph-1-1.md

```diff
@@ -1,46 +1,65 @@
 +++
 date = "2017-03-20T22:25:17+11:00"
-title = "Migrate to Dgraph v1.1"
+title = "Migrate from Dgraph v1.0"
 weight = 11
 [menu.main]
     parent = "migration"
 +++
 
+This document summarizes major changes in Dgraph database that you should be
+aware of as you migrate your data and client applications from Dgraph v1.0 to
+newer Dgraph versions.
+
 ## Schema types: scalar `uid` and list `[uid]`
 
-The semantics of predicates of type `uid` has changed in Dgraph 1.1. Whereas before all `uid` predicates implied a one-to-many relationship, now a one-to-one relationship or a one-to-many relationship can be expressed.
+The semantics of predicates of type `uid` has changed since Dgraph v1.0. 
+In Dgraph v1.0 all `uid` predicates implied a one-to-many relationship, but now
+you can express a one-to-one relationship or a one-to-many relationship. The
+following syntax example demonstrates both types of relationships:
 
-```
+```text
 friend: [uid] .
 best_friend: uid .
 ```
 
-In the above, the predicate `friend` allows a one-to-many relationship (i.e a person can have more than one friend) and the predicate best_friend can be at most a one-to-one relationship.
+In the above example, the predicate `friend` allows a one-to-many relationship
+(i.e a person can have more than one friend) and the predicate `best_friend` 
+allows a one-to-one relationship.
+
+The new `uid` syntax is consistent with that used for other types. For example,
+`string` indicates a  single-value string and `[string]` represents many strings.
 
-This syntactic meaning is consistent with the other types, e.g., `string` indicating a single-value string and `[string]` representing many strings. This change makes the `uid` type work similarly to other types.
+To migrate existing schemas from Dgraph v1.0 to newer Dgraph versions, update
+the schema file from an export so all predicates of type `uid` are changed to
+`[uid]`. Then, use the updated schema when loading data into Dgraph v1.1. 
 
-To migrate existing schemas from Dgraph v1.0 to Dgraph v1.1, update the schema file from an export so all predicates of type `uid` are changed to `[uid]`. Then use the updated schema when loading data into Dgraph v1.1. For example, for the following schema:
+For example, you could start with the following schema in Dgraph v1.0:
 
+<!-- NOSUBMIT comment for reviewers: OK to change the syntax blocks here to '```graphql' ? --> 
 ```text
```

Comment:
Comment for reviewers: Is it OK to change the syntax blocks here to '```graphql' ?

Reply:
mmm. that does seem a graphql syntax, I would live it as a text box